runtime/libs/tflite/src/RandomTestRunner.cpp

```diff
@@ -36,6 +37,60 @@ namespace tflite
 
 using namespace std::placeholders;
 
+class CopyInputInitializer
```

Comment:
@hseok-oh (optional) What about getting `CopyInputInitializer` out of `RandomTestRunner.cpp` like `RandomInputInitializer`? It can have its own module (`CopyInputInitializer.[h,cpp]`) or could be merged into `RandomInputInitializer` (in this case, the name would be good to be renamed (`RandomInputInitializer` →`InputInitializer`).

Reply:
@glistening Different with draft, I didn't move `CopyInitializer` out of `RandomTestRunner.cpp` because it is used in `RandomTestRunner.cpp` only. If it is used by other module, we can move this.